data/strings/types_strings.txt

```diff
@@ -4923,7 +4923,7 @@
     tr = Bisiklet Bariyeri
     uk = Велосипедна перешкода
 
-  [type.barrier.ditch]
+  [type.waterway.ditch]
     en = Moat
```

Comment:
A moat is usually quite wide, deep and used as a defense structure, right?
While waterway-ditch is usually a small trench for draining.

So maybe something like a "Drainage Ditch" would be better?

Overall looking at e.g. slavic translations I can't really say which one is with water and which one is dry :)
I bet there is a similar issue for many other languages (and it probably explains why this translation inconsistency was unnoticed for a long time).

So probably it would be best first to devise better EN translations and then ask the translations team to update other languages.

Reply:
Ah true, the correct type for moat would be `type.natural.water.moat`, but we don't include it & there are less than 2000 uses, so i'll leave the translation for the future.